### ISSUE TITLE
Card stopped working with Environment Canada 

### ISSUE THREAD

**Issue by HomeWiseAdmin** [bramkragten/weather-card#172]
Hello,

After the recent update on the Environment Canada integration by HA team where they have somehow put everything in one entity (Daily and Hourly), the weather card basically doesn't show anything mow, only the current state. I am running the HA April 2024 release as well as the latest Weather-Card. Could you please look into it  

Here is a link and more information:

https://github.com/home-assistant/core/pull/112447

Breaking change
The Environment Canada integration previously created two entities for each configured location, one entity which provided daily weather forecasts and one entity which provided hourly forecasts. The Environment Canada integration now only creates a single entity which provides both daily and hourly weather forecasts.

**Comment by TheLordVader** [comment]
Yes! Same for me with SMHI integration by HA. Started as of 2024.04 release.

**Comment by arsenicks** [comment]
That's a duplicate of #170, I think you can close this one. This is not related to a specific weather source.

Workaround is to use beta version for now: https://github.com/bramkragten/weather-card/issues/170#issuecomment-2038572364

This worked for me at least. I had to change the entity from weather.location_hourly to weather.location and enable hourly in the ui.

**Comment by stamandr** [comment]
Using the beta version doesn't work for me.  See pics Old version with Core 2024.3.3 (top) New version (bottom) with Core 2024.4.3

![image](https://github.com/bramkragten/weather-card/assets/51379504/6ae1ce70-cbcb-4ac5-af13-6eedd80ef37b)

![image](https://github.com/bramkragten/weather-card/assets/51379504/c998f73d-f363-40eb-9349-9a8b0391f77a)

**Comment by parautenbach** [comment]
<img width="615" alt="Screenshot 2024-05-04 at 08 17 29" src="https://github.com/bramkragten/weather-card/assets/743320/ad275022-ba98-4d8d-9840-8ac7132e9f53">

**Comment by stamandr** [comment]
Same version
